main.py

```diff
@@ -2,25 +2,36 @@
 
 app = Flask(__name__)
 
-# this url route displays some html tags on the home page
 @app.route("/")
 def home():
-    return "<h1>Image Analysis System</h1> <p>Hello world!</p>"
+    return render_template("home.html")
+
+@app.route("/events")
+def events():
+    return render_template("events.html")
+
+@app.route("/cameras")
+def cameras():
+    return render_template("cameras.html")
+
+@app.route("/train")
+def train():
+    return render_template("train.html")
 
 # this url route will take the input passed in and display it
-@app.route("/<name>/")
-def testURLInput(name):
-    return f"Hello {name}!"
+# @app.route("/<name>/")
+# def testURLInput(name):
+#     return f"Hello {name}!"
 
 # this url route will redirect to the home page
-@app.route("/admin/")
-def redirectWithoutParam():
-    return redirect(url_for("home"))
+# @app.route("/admin/")
+# def redirectWithoutParam():
+#     return redirect(url_for("home"))
 
 # this url route will redirect to the /name page with a specific param
-@app.route("/admin_param/")
-def redirectWithParam():
-    return redirect(url_for("testURLInput", name="Dr. Huda"))
+# @app.route("/admin_param/")
+# def redirectWithParam():
+#     return redirect(url_for("testURLInput", name="Dr. Huda"))
 
 if __name__ == "__main__":
-    app.run()
+    app.run(debug=True)
```

Comment:
The `debug=True` line prevents the need to always kill and restart the server in between changes.